conans/client/conf/__init__.py

```diff
@@ -40,28 +40,28 @@
                       "11.0", "11.1", "11.2", "11.3", "11.4", "12.0", "12.1", "12.2", "12.3", "12.4",
                       "13.0", "13.1", "13.2", "13.3", "13.4", "13.5", "13.6", "13.7",
                       "14.0", "14.1", "14.2", "14.3", "14.4", "14.5", "14.6", "14.7", "14.8",
-                      "15.0", "15.1", "15.2", "15.3", "15.4"]
+                      "15.0", "15.1", "15.2", "15.3", "15.4", "15.5", "15.6", "16.0"]
```

Comment:
There is already Xcode 14.1 Beta in the works which is targeting iOS 16.1, TV OS 16.1. Should these be added now?
https://developer.apple.com/documentation/xcode-release-notes/xcode-14_1-release-notes
https://xcodereleases.com

Reply:
Yes. I am downloading new xcodes an will repor what version do they report.

Reply:
14.1 beta:

> Apple clang version 14.0.0 (clang-1400.0.29.201)
> Target: x86_64-apple-darwin21.6.0

Reply:
I've added commit with new iOS/watchOS/tvOS versions. Do we want to add all macOS minor versions?
There are too many of them...